x-pack/filebeat/module/rabbitmq/log/manifest.yml

```diff
@@ -0,0 +1,15 @@
+module_version: 1.0
+
+var:
+  - name: paths
+  # TODO: ideally, respect ENV values from https://www.rabbitmq.com/relocate.html#locations
```

Comment:
I would suggest to skip this for now. @exekias Do we have something similar somewhere else? @kaiyan-sheng In AWS module we also use env variables, but not sure if that works in combination with the manifest.

Reply:
uhm I don't know if ENV vars are supported here, we definitely use them in some modules of Metricbeat

Reply:
6864ca7 added support for `RABBITMQ_LOGS` env var